### PR TITLE
Improve plugins development

### PR DESCRIPTION
**Pull request by joaquimrocha**
This PR allows to build local plugins + changes the default plugins dir.

### CODE REVIEW

File: frontend/src/redux/reducers/ui.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ function reducer(state = INITIAL_STATE, action: Action) {
     }
     case UI_DETAILS_VIEW_SET_HEADER_ACTION: {
       const headerActions = {...newFilters.views.details.headerActions};
-      headerActions[action.actionName as string] = action.action;
+      headerActions[action.name as string] = action.action;
```
Comment (ashu8912) [inline frontend/src/redux/reducers/ui.tsx:87]: This change doesn't seem to belong here

Comment (joaquimrocha) [reply frontend/src/redux/reducers/ui.tsx]: Good catch. I have also added that to another PR. This was mistake.

**Review by ashu8912 — APPROVED** [review]
Tested and it looks fine to me

**Comment by ashu8912** [comment]
@joaquimrocha we can merge this one

**Pull request merged**